sfge/src/main/java/com/salesforce/rules/OccurrenceInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.salesforce.rules;
+
+import com.salesforce.config.UserFacingMessages;
+
+/** Internal representation of an occurrence info. Originally from MMSL Info Utils. */
+public class OccurrenceInfo {
```

Comment:
Something about having this as a top-level class sits weird with me. I'm not immediately sure where else it belongs, though.